### DISCUSSION TITLE
Strange behavior after connection

### DISCUSSION THREAD

**Discussion by Foggy211** [BimmerGestalt/AAIdrive#737] [Q&A]
Hello,

I have à F36 430dxdrive years 2015, with NBT Evo ID.4

I have done everything necessary and prerequisites to establish the connection. (USB)
All is ok, and menu displayed a few music apk on screen.

But strangely afterwards, as soon as I select an application (for example streaming), it leads me to a screen named "detailed vehicle info" in english (im french, with french menu)

impossible to launch any application, it systematically leads me to this screen

Here are the photos for more understanding :

**Before connection :** 

![IMG_20230911_135613](https://github.com/BimmerGestalt/AAIdrive/assets/144688798/3f70c8c2-8359-44fb-b9f5-79678435c479)

**After connection:**

![IMG_20230911_140007](https://github.com/BimmerGestalt/AAIdrive/assets/144688798/f3651240-a5cb-44ac-b891-8f079177f5d0)

**Screen after selection of an APP :**

![IMG_20230911_140021](https://github.com/BimmerGestalt/AAIdrive/assets/144688798/a22ecfd6-6bd8-453f-a1a6-37b2c033d021)

✌️

**Comment by hufman** [comment]
The various music apps, as well as the Phone Notifications app and the new Vehicle Information app, are technically placeholder icons in the car api. Clicking them triggers the owning app to bring itself forward (Audioplayer, the unlabeled book icon, and the unlabeled empty icon).
I think ID4 has a bug in this foreground step, back from when I still had an ID4 car, and I think it might only show the most recently-shown app.
Does the behavior change when you manually open the Audioplayer app and then use the other music apps to switch input sources?
Should AAIdrive not show this Vehicle Information icon at all? Since it does not reliably trigger the new app anyways, maybe it's better to only show the Audioplayer icons?
